docs/source/documentation/adrs/adr-005-use-aws-secrets-manager-for-secrets.html.md.erb

```diff
@@ -0,0 +1,42 @@
+---
+owner_slack: "#data-platform-notifications"
+title: ADR-005 Use AWS Secrets Manager for Secrets
+last_reviewed_on: 2023-04-28
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Status
+
+✅ Accepted
+
+## Context
+
+The Data Platform team will need a way to store secrets securely. There are [several methods](https://security-guidance.service.justice.gov.uk/secrets-management/#application--infrastructure-secrets) currently used across the MoJ, including [Secrets Manager](https://aws.amazon.com/secrets-manager/), [Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html), [1Password](https://operations-engineering.service.justice.gov.uk/documentation/services.html#services), [Git-Crypt](https://github.com/AGWA/git-crypt) and [GitHub Secrets.](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+
+We want to adhere to [MoJ Security Guidance](https://security-guidance.service.justice.gov.uk/secrets-management/#application--infrastructure-secrets) and align with other [Hosting and Platform teams.](https://technical-guidance.service.justice.gov.uk/documentation/standards/hosting.html)
+
+## Decision
+
+We are proposing to use [Secrets Manager](https://aws.amazon.com/secrets-manager/) for secrets management. We can use it for our GitHub actions as seen [here.](https://github.com/ministryofjustice/data-platform/blob/main/.github/workflows/platform-pagerduty-on-call.yml#L48-L54)
+
+[AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) can be used to store non secret information e.g. environment parameters
+
+## Consequences
+
+### General consequences
+
+* All secrets will be stored in Secrets Manager
```

Comment:
Would it be possible to articulate what advantage in terms of keeping secret info secret does SM have over PS?